test/orange/OrangeGeoTestBase.cc

```diff
@@ -125,43 +122,32 @@ void OrangeGeoTestBase::build_geometry(TwoVolInput inp)
 {
     CELER_EXPECT(!params_);
     CELER_EXPECT(inp.radius > 0);
-    UnitInput input;
 
-    BBox bbox = {{-inp.radius, -inp.radius, -inp.radius},
-                 {inp.radius, inp.radius, inp.radius}};
-
-    auto inf = std::numeric_limits<real_type>::infinity();
-
-    BBox inf_bbox = {{-inf, -inf, -inf}, {inf, inf, inf}};
+    UnitInput input;
+    input.label = "two volumes";
+    input.bbox = {{-inp.radius, -inp.radius, -inp.radius},
+                  {inp.radius, inp.radius, inp.radius}};
 
-    {
-        // Insert surfaces
-        SurfaceInputBuilder insert(&input.surfaces);
-        insert(Sphere({0, 0, 0}, inp.radius), Label("sphere"));
-    }
-    {
-        // Insert volumes
+    input.surfaces = {Sphere({0, 0, 0}, inp.radius)};
+    input.surface_labels = {Label("sphere")};
+    input.volumes = [] {
+        std::vector<VolumeInput> result;
         VolumeInput vi;
         vi.faces = {LocalSurfaceId{0}};
 
         // Outside
         vi.logic = {0};
         vi.label = "outside";
-        vi.bbox = inf_bbox;
-        input.volumes.push_back(vi);
+        vi.bbox = BBox::from_infinite();
+        result.push_back(vi);
 
         // Inside
         vi.logic = {0, logic::lnot};
         vi.label = "inside";
-        vi.bbox = bbox;
-        input.volumes.push_back(vi);
-    }
-
-    // Save bbox
-    input.bbox = {{-inp.radius, -inp.radius, -inp.radius},
-                  {inp.radius, inp.radius, inp.radius}};
-
-    input.label = "two volumes";
+        vi.bbox = BBox::from_infinite();
```

Comment:
Was the change to this bounding box intentional?

Reply:
Nope, great catch!